test/image/Dockerfile

```diff
@@ -31,14 +28,16 @@ RUN curl -sSLO https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.t
     tar -zxvf v${BATS_VERSION}.tar.gz && \
     bash bats-core-${BATS_VERSION}/install.sh /usr/local
 
-# Install ORAS
-RUN curl -SsLO https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_${arch}.tar.gz && \
+# Install oras
+RUN curl -SsLO https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_${TARGETARCH}.tar.gz && \
     mkdir -p oras-install/ && tar -zxf oras_${ORAS_VERSION}_*.tar.gz -C oras-install/ && \
     mv oras-install/oras /usr/local/bin/ && rm -rf oras_${ORAS_VERSION}_*.tar.gz oras-install/
 
-# Install yq and jq
-RUN curl -LsS https://github.com/mikefarah/yq/releases/download/v${YQ_VERSION}/yq_linux_${arch} -o /usr/local/bin/yq \
+# Install yq
+RUN curl -LsS https://github.com/mikefarah/yq/releases/download/v${YQ_VERSION}/yq_linux_${TARGETARCH} -o /usr/local/bin/yq \
     && chmod +x /usr/local/bin/yq
+
+# Install jq
```

Comment:
Why do we need to install from github releases and apt-get install again later?

Reply:
> Why do we need to install from github releases and apt-get install again later?

jq is installed through apt get, I didn't change it soon

Reply:
my bad I thought that was yq. need coffee :)